### ISSUE TITLE
Replace Sonic with Elasticsearch

### ISSUE THREAD

**Issue by tvolk131** [tvolk131/fdr-finder#19]
I chose Sonic as the search backend because of its simplicity and low resource usage. However, while the search results it provides are adequate, its autocomplete functionality is not. As far as I can tell, autocomplete suggestions can't operate on more than one word, meaning that suggestions are unable to use the context of the entire search term as the user is typing. Elasticsearch would fix this problem with the use of a [phrase suggestor](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-suggesters.html#phrase-suggester). Let's swap out Sonic in favor of Elasticsearch for the search backend.

**Comment by tvolk131** [comment]
After doing some investigating, I discovered [Meilisearch](https://www.meilisearch.com/). This could be a good alternative to Elasticsearch, since it claims to have the ability to show rank-ordered and context-aware results. However, it currently does not support autocomplete suggestions, so if we used it for search then we'd still probably want to use Elasticsearch for autocomplete results, which makes it a less appealing option. Still, worth considering since it is much simpler to setup than Elasticsearch would be.
